### ISSUE TITLE
Fix code syntax highlighting colors

### ISSUE THREAD

**Issue by ElectricRCAircraftGuy** [ElectricRCAircraftGuy/ElectricRCAircraftGuy.github.io#16]
I don't like the dark syntax highlighting background and colors. Fix them to be light, like GitHub or Stack Overflow.

See: 
1. https://github.com/mmistakes/minimal-mistakes/issues?q=is%3Aissue+author%3A%40me+is%3Aclosed
    1. In particular: https://github.com/mmistakes/minimal-mistakes/issues/2770
1. https://github.com/mmistakes/minimal-mistakes/pulls?q=is%3Apr+author%3A%40me+is%3Aclosed
   1. In particular: https://github.com/mmistakes/minimal-mistakes/pull/2771

**Comment by ElectricRCAircraftGuy** [comment]
Note: commit 464b72d7eb92bd5e0dcb525e6c7f9ca05b8fcbbc made code format between backticks get properly highlighted even when within headings.

See:
```bash
git difftool 464b72d7eb92bd5e0dcb525e6c7f9ca05b8fcbbc~..464b72d7eb92bd5e0dcb525e6c7f9ca05b8fcbbc
```

**Comment by ElectricRCAircraftGuy** [comment]
From: http://localhost:4001/ydotool-tutorial/ and https://gabrielstaples.com/ydotool-tutorial/

I super hate it! I hate this dark color so much. I want it to look more like Stack Overflow instead.

![image](https://github.com/ElectricRCAircraftGuy/ElectricRCAircraftGuy.github.io/assets/6842199/16463795-18c2-450b-9d06-0ff7c3ac4500)

**Comment by ElectricRCAircraftGuy** [comment]
Fixed by the commit above!: 9315052f82aab14bac5101f9b18470aba779131c

Much better! Here's what it looks like now!:

![image](https://github.com/ElectricRCAircraftGuy/ElectricRCAircraftGuy.github.io/assets/6842199/b8bb6ee2-0438-415e-96c4-27f47da08cb3)

**Comment by ElectricRCAircraftGuy** [comment]
Done.
